csharp/src/Ice/ACM.cs

```diff
@@ -105,204 +162,125 @@ internal void Destroy()
             {
```

Comment:
I don't think we need to clear the set of _connections here... no circular dependencies issues with C#. We could simplify Destroy to call Dispose on the timer.

Reply:
Fixed